pkg/scheduler/api/resource_info.go

```diff
@@ -399,6 +399,38 @@ func (r *Resource) LessInSomeDimension(rr *Resource) bool {
 	return false
 }
 
+// LessEqualInSomeDimension returns true if there exists any dimension whose resource amount in r is less than or equal with that in rr.
+// Otherwise returns false.
+// @param defaultValue "default value for resource dimension not defined in ScalarResources. Its value can only be one of 'Zero' and 'Infinity'"
+func (r *Resource) LessEqualInSomeDimension(rr *Resource, defaultValue DimensionDefaultValue) bool {
```

Comment:
prefer to rename to `LessEqual` & `LessEqualPartly` :)

Reply:
> prefer to rename to `LessEqual` & `LessEqualPartly` :)

Sounds great! That will be brief and to the point. Let me update it.

Reply:
> prefer to rename to `LessEqual` & `LessEqualPartly` :)

Sounds great. That will be brief and to the point.

Reply:
Updated